locale/en/en.cfg

```diff
@@ -5,9 +5,9 @@ AdvancedArtilleryRemotesContinued=Advanced Artillery Remotes Continued
 artillery-cluster-remote=Artillery Cluster Remote
 artillery-discovery-remote=Artillery Discovery Remote
 
-[recipe-name]
-artillery-cluster-remote=Artillery Cluster Remote
-artillery-discovery-remote=Artillery Discovery Remote
+[shortcut-name]
+give-artillery-cluster-remote-artillery-shell=Give Artillery Cluster Remote
```

Comment:
Hm... Not sure if the `-artillery-shell` suffix should be here, but I'll have to poke around no how the localisation will behave with the ability to have multiple remotes for different ammo types.

Reply:
Hm... I'll see if I can change it and see what happens. Pretty sure the code should be able to work without that suffix unless I am mistaken.